build/jest-framework-setup.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-env node */
+import {configure} from 'enzyme';
```

Comment:
What if people don't use enzyme in their tests?

Reply:
I don't think it's necessary, but I think we should probably enable by default? We can add a config to turn this off though.